torch/csrc/jit/codegen/cuda/parser.cpp

```diff
@@ -1560,7 +1560,57 @@ class IrParser {
             nullptr);
       }
     }
+    {
+      if (isOptionEnabled(EnableOption::GraphOp)) {
+        auto ptr_op = getOperatorForLiteral(
+            "aten::gather(Tensor self, int dim, Tensor index, *, bool sparse_grad=False) -> Tensor");
+        REGISTER_PARSE_RULE(
+            ptr_op,
+            {
+              MemoryFormat format;
+              std::list<Val*> list_val;
+              std::tie(format, list_val) = getPWFormatValues(
+                  c10::nullopt,
```

Comment:
I don't think the parsing rule is permutation agnostic. Let's change this from `c10::nullopt` -> `MemoryFormat::Contiguous()`

Reply:
Noticed that this is also the case for `index_select`, which I missed in my last review. Can we also update that. :stuck_out_tongue_winking_eye:

Reply:
Done in new commit